docs/references/events.mdx

```diff
@@ -562,3 +562,82 @@ This [Event](/workflows#event) type indicates that a [Workflow Execution](/workf
+| schedule_to_close_timeout        | Schedule-to-close timeout for this operation. Indicates how long the caller is willing to wait for operation completion. Calls are retried internally by the server.                                                                                                                                      |
+| nexus_header                     | Header to attach to the Nexus request. Note these headers are not the same as Temporal headers on internal activities and child Workflows, these are transmitted to Nexus operations that may be external and are not traditional payloads.                                                               |
+| workflow_task_completed_event_id | The `WORKFLOW_TASK_COMPLETED` event that the corresponding ScheduleNexusOperation command was reported with.                                                                                                                                                                                              |
+| request_id                       | A unique ID generated by the History Service upon creation of this event. The ID will be transmitted with all Nexus StartOperation requests and is used as an idempotency key.                                                                                                                            |
+| endpoint_id                      | Endpoint ID as resolved in the endpoint registry at the time this event was generated. This is stored on the event and used internally by the server in case the endpoint is renamed from the time the event was originally scheduled.                                                                    |
+
+### NexusOperationStarted
+
+This Event type indicates that an Nexus Operation Execution was started.
+The handler’s SDK Worker picked up the Nexus Operation and started processing the Nexus Operation invocation.
+
+| Field              | Description                                                                                                                                 |
+| :----------------- | :------------------------------------------------------------------------------------------------------------------------------------------ |
+| scheduled_event_id | The ID of the `NEXUS_OPERATION_SCHEDULED` event this task corresponds to.                                                                   |
+| operation_id       | The operation ID returned by the Nexus handler in the response to the StartOperation request. This ID is used when canceling the operation. |
+| request_id         | The request ID allocated at schedule time.                                                                                                  |
+
+### NexusOperationCompleted
+
+This Event type indicates that the Nexus Operation has completed.
+The handler’s SDK client has picked up and successfully completed the Nexus Operation.
```

Comment:
See comments above re: sync vs. async Nexus ops. This event can happen either:
a) Sync Nexus Op: after single Nexus request/reply between caller and handler Nexus machinery
b) Async Nexus Op: after (a) Nexus start operation request -> (b) started -> (c) completed async

```suggestion
The caller's workflow history records the result of a successful Nexus Operation with this event for synchronous and asynchronous Nexus Operations. 
```